code/modules/mob/living/carbon/xenomorph/castes/shrike/abilities_shrike.dm

```diff
@@ -232,6 +232,7 @@
 		KEYBINDING_NORMAL = COMSIG_XENOABILITY_PSYCHIC_CURE,
 	)
 	var/heal_range = SHRIKE_HEAL_RANGE
+	var/do_after_flags = IGNORE_TARGET_LOC_CHANGE
```

Comment:
why is this even a var? All the do_after's are separate so can't you just specify them there?